src/test/unit_tests/index.js

```diff
@@ -52,9 +52,6 @@ require('./test_s3_list_objects');
 require('./test_nb_native_b64');
 require('./test_bucket_chunks_builder');
 require('./test_mirror_writer');
-require('./test_namespace_fs');
```

Comment:
Do we really need to move also the test_namespace_fs into the sudo index? 
my concern here is that locally we will still use `npm run tests` and not `npm run test:local` which means that we will skip this.

Reply:
I agree that now we have 2 options of running tests and it's confusing,
but since we call namespace_fs functions and these functions need to set_uid() and set_gid() correctly, I believe that it's needed for future tests (currently, there is no permissions check in test_namespace_fs so it still passes without root permissions).

Reply:
I think that for now, as the suit does not need permissions, we can leave it in the main index. 
We can move it later if we will add permissions to that suit 
WDYT?

Reply:
I think the tests are not running properly without root permissions so running them incorrectly can cause future issues that we want to avoid, @jeniawhite WDYT?

Reply:
fixed.